### PR TITLE
Rebased and Finished Feature

### PR DESCRIPTION
**Pull request by ify1101**
This PR was created to make changes and update feature

### CODE REVIEW

File: lib/ui/views/new_dish/new_dish_view.dart [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                    AppImages.burgerLog,
+                    width: 24.7.w,
+                    height: 24.h,
+                  ),
+                ),
+              ),
+      title:Text(S.current.create_a_dish,
+                style: typography?.titleBold16?.copyWith(color: palette?.gray11)) ,     
+      ),
+      body: SingleChildScrollView(
```
Comment (moak13) [inline lib/ui/views/new_dish/new_dish_view.dart:56]: Let's try as much as possible to reduce the amount of widgets we are calling. For your scrolling, you can make use of a `ListView` which gives you `padding` automatically than what you have done here.

**Review by moak13 — CHANGES_REQUESTED** [review]
Awesome work, see the concern below.

File: lib/ui/views/new_dish/new_dish_view.dart (hunk 1 above)
Comment (ikennaumeh) [reply lib/ui/views/new_dish/new_dish_view.dart:56]: Tricky thing here is that while list view is good, single child and column is better when using forms. List view "deletes" widget not in view, so imagine there's a validation error somewhere up the widget tree, and you scroll bk up to find it, you won't see it

**Review by ikennaumeh — APPROVED** [review]
Lgtm

**Review by moak13 — APPROVED** [review]

**Pull request merged**